### ISSUE TITLE
Unable to catkin build 

### ISSUE THREAD

**Issue by ZhiWei20011** [ctu-mrs/object_detect#2]
Hi, I am planning to add this object color segmentation detection onto my drone but whenever after I git clone the code into my mrs_workspace/src directory, I cant seem to be able to catkin build successfully. Did I miss out or do wrongly for any steps for example cloning into the wrong directory. Thanks in advanced!
![Screenshot from 2021-07-15 18-23-08](https://user-images.githubusercontent.com/85549044/125717830-3353ed55-87cd-469a-8656-aee9f9bd249d.png)

**Comment by matemat13** [comment]
According to the error, CMake was unable to find OpenCV 4.2. It seems you only have 3.2. This is probably because you're still on Ubuntu 18.04 and ROS Melodic. This package was updated to run on Ubuntu 20.04 + ROS Noetic (as is stated in the Readme) which comes with OpenCV 4.2 and is not backward compatible with 18.04 + Melodic (which has OpenCV 3.2). You can try instaling OpenCV 4.2, which may be sufficient to compile this package, but maybe there will be some other incompatibility, I haven't tried it.

tl;dr: Try installing OpenCV 4.2 and if it still doesn't work, upgrade to Ubuntu 20.04 and ROS Noetic.

**Comment by matemat13** [comment]
closing due to inactivity
